draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -909,6 +913,52 @@ previous section.
 {::include ./examples/ML-DSA-87.crt.txt}
 ~~~
 
+
+## Example Inconsistent Private Keys {#example-bad}
+
+<aside markdown="block">
+  WARNING: These private keys are pruposely bad do not use them in
+  production systmes.
+</aside>
+
+The following examples demonstrate inconsistent seed and expanded private keys.
+
+### ML-DSA Inconsistent Seed and Expanded Private Keys
+
+Three `ML-DSA-44-PrivateKey both CHOICE` examples of inconsistent seed and
+expanded private keys follow:
+
+1. The first is for inconsistent `seed` and `expandedKey`.
+
+2. The second is an `expandedkey` for which the recomputed public key
+   hash t(r) fails to match the private key.
+
+3. The third is an `expandedKey` for which the recomputed public t(0)
+   fails to match the private key, i.e., the two t(r) do not match.
+
+The second and third mismatches would not be detected by implementations
+that do not regenerate the public key from the private key, or neglect to
+then check consistency of t(r) or t(0).
```

Comment:
In contrast to the other two, we don't have any normative language (yet, #116) on whether an implementation should accept or reject the third example.

Reply:
Editorial: $`tr`$ is not $`t(r)`$ and $`t_0`$ is subscripted, not parameterised by `0`.  So, if MATHML is to be avoided perhaps, just \`tr\` and \`t0\` or \`t_0\`?